src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -1137,5 +1137,401 @@ public void WithParsing19()
             }
             EOF();
         }
+
+        [Theory]
+        [CombinatorialData]
+        public void Base_01(
+            [CombinatorialValues(true, false)] bool withDataKeyword,
+            [CombinatorialValues("class", "struct")] string typeKeyword,
+            [CombinatorialValues(true, false)] bool withParameters,
+            [CombinatorialValues(true, false)] bool withBaseArguments,
+            [CombinatorialValues(true, false)] bool withBody)
```

Comment:
Are the `[CombinatorialValues]` attributes needed for `bool` parameters?

Reply:
> Are the [CombinatorialValues] attributes needed for bool parameters?

I don't know, its presence works great.

---
In reply to: [434100804](https://github.com/dotnet/roslyn/pull/44733#discussion_r434100804) [](ancestors = 434100804)